SpatialGDK/Source/SpatialGDKEditor/Private/SnapshotGenerator/SpatialGDKEditorSnapshotGenerator.cpp

```diff
@@ -49,10 +49,10 @@ bool CreateSpawnerEntity(Worker_SnapshotOutputStream* OutputStream)
 	ComponentWriteAcl.Add(SpatialConstants::ENTITY_ACL_COMPONENT_ID, SpatialConstants::UnrealServerPermission);
 	ComponentWriteAcl.Add(SpatialConstants::PLAYER_SPAWNER_COMPONENT_ID, SpatialConstants::UnrealServerPermission);
 
-	Components.Add(improbable::Position(improbable::Origin).CreatePositionData());
-	Components.Add(improbable::Metadata(TEXT("SpatialSpawner")).CreateMetadataData());
-	Components.Add(improbable::Persistence().CreatePersistenceData());
-	Components.Add(improbable::EntityAcl(SpatialConstants::ClientOrServerPermission, ComponentWriteAcl).CreateEntityAclData());
+	Components.Add(SpatialGDK::Position(SpatialGDK::Origin).CreatePositionData());
```

Comment:
```suggestion
	Components.Add(Position(Origin).CreatePositionData());
```
This as well as other instances of `SpatialGDK` in this file seem unnecessary since we are `using namespace SpatialGDK`.

Reply:
spatialGDKsnapshot